oceannavigator/frontend/src/components/OceanNavigator.jsx

```diff
@@ -167,7 +180,7 @@ class OceanNavigator extends React.Component {
     }
 
     if (key === "dataset_1") {
-      this.setState(value);
+      this.setState({dataset_1: value});
```

Comment:
Great catch 🤦

Reply:
All good, took an embarrassingly long time to realize why the right map options were updating the left map haha

Reply:
Don't sweat it lol, I guarantee it would have taken me just as long to see that.